libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKMapper.kt

```diff
@@ -94,6 +94,11 @@ fun asCheckMessageRequest(reqData: ReadableMap): CheckMessageRequest? {
     return null
 }
 
+fun asReverseSwapFeesRequest(reqData: ReadableMap): ReverseSwapFeesRequest? {
+    val sendAmountSats = reqData.getDouble("sendAmountSats")
+    return ReverseSwapFeesRequest(sendAmountSats)
+}
```

Comment:
If sendAmountSat is optional, getDouble will return 0 if not set. 
asReverseSwapFeesRequest always returns a ReverseSwapFeesRequest
```
fun asReverseSwapFeesRequest(reqData: ReadableMap): ReverseSwapFeesRequest {
    val sendAmountSats = if (hasNonNullKey(reqData, "sendAmountSats")) reqData.getDouble("sendAmountSats") else null
    return ReverseSwapFeesRequest(sendAmountSats)
}
```